### PR TITLE
Finish chapter 6

### PR DESCRIPTION
**Pull request by quanla-0022**
image

### CODE REVIEW

**Comment by quanla-0022** [comment]
ready

**Comment by quanla-0022** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  before_save{self.email = email.downcase}
```
Comment (thanhhungchu95) [inline app/models/user.rb]: dùng private method thay vì block trực tiếp nak

**Comment by quanla-0022** [comment]
ready

**Pull request merged**